tests/unit/examples/test_notebooks_gpu.py

```diff
@@ -124,10 +124,3 @@ def test_dkn_quickstart(notebooks, output_notebook, kernel_name):
         kernel_name=kernel_name,
         parameters=dict(epochs=1, batch_size=500),
     )
-
-
-@pytest.mark.notebooks
-@pytest.mark.gpu
-def test_sasrec_single_node_runs(notebooks, output_notebook, kernel_name):
-    notebook_path = notebooks["sasrec_quickstart"]
-    pm.execute_notebook(notebook_path, output_notebook, kernel_name=kernel_name)
```

Comment:
quick question, is there a similar test in smoke or integration?

Reply:
Yes, there is one in integration (the same notebook). I removed this one because it took about 90 minutes to complete.

Reply:
The integration test is quite slow (2h 20 mins).